### PR TITLE
v0.11.0

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Default: `'JST'`
 The namespace in which the precompiled templates will be assigned.  *Use dot notation (e.g. App.Templates) for nested namespaces or false for no namespace wrapping.*  When false with `amd` or `commonjs` option set `true`, templates will be returned directly from the AMD/CommonJS wrapper.
 
 Example:
-```js
+```javascript
```
Comment (XhmikosR) [inline README.md]: What happened here with the docs? Why did you revert my changes?

Comment (vladikoff) [inline README.md]: thought ```js was still broken, we can fix it later or just leave it

Comment (vladikoff) [inline README.md]: also let's remove the trailing spaces, they get in the way and the formatting is not suffering too much

Comment (XhmikosR) [inline README.md]: I honestly don't see how they get in the way. Nor how `js` was broken.

I just tried being consistent. And Markdown needs two trailing spaces for line breaks as you know.

Formatting looks bad without the line breaks. I'd advice you to stick with the current scheme. If you really don't like the line breaks we should all agree to another solution. Maybe using `*` (ul).

Comment (vladikoff) [inline README.md]: ok we can revert that in the future, I have no preference in the formatting

**Pull request merged**